pkgs/by-name/aw/awatcher/package.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, openssl
+, pkg-config
+, gnomeSupport ? true
+, kwinSupport ? true
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "awatcher";
+  version = "0.2.7";
+
+  src = fetchFromGitHub {
+    owner = "2e3s";
+    repo = "awatcher";
+    rev = "v${version}";
+    hash = "sha256-e65QDbK55q1Pbv/i7bDYRY78jgEUD1q6TLdKD8Gkswk=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ openssl ];
+
+  buildNoDefaultFeatures = true;
+  buildFeatures = [ ]
+    ++ lib.optional gnomeSupport "gnome"
+    ++ lib.optional kwinSupport "kwin_window";
+
```

Comment:
```suggestion
```

Reply:
Why remove this?

Reply:
If there are no extra dependencies, we might as well always build the full package